### DISCUSSION TITLE
Support for :slug in the page URL

### DISCUSSION THREAD

**Discussion by sellscompany** [OFFLINE-GmbH/oc-boxes-support#35] [Ideas and feature requests]
Are there plans to support :slug in the page URL?

![image](https://user-images.githubusercontent.com/40325840/199558218-23a47842-2b49-47d7-bb56-e3868e528414.png)

**Comment by tobias-kuendig** [comment]
How/where would you want to utilize the `:slug` parameter afterwards?

**Comment by sellscompany** [comment]
To use Boxes to build detailed pages templates

**Comment by tobias-kuendig** [comment]
Did you see that you can integrate the Boxes editor in Tailor?

https://docs.boxes.offline.ch/use-cases/usage-in-plugins.html#use-the-boxes-editor-in-tailor

The scope of the Boxes Editor is currently aimed at static content pages. Building a whole blog in the Editor is possible in theory, but would also introduce a lot of complexity. 

Maybe a Tailor based solution would work better for this use-case?

**Comment by sellscompany** [comment]
I understand that it is difficult now. Are you planning this expansion in the future?

**Comment by tobias-kuendig** [comment]
This feature is not planned at the moment. I'll leave the issue open to discuss use-cases. If there is enough interest, we can implement this feature.

**Comment by PubliAlex** [comment]
@sellscompany I have some difficulties to imagine the use case of this. If you need a :slug parametre in your page, you need a model to handle that dynamic content.

As you can integrate boxes as a field in a model (tobias, you talk about tailor, but boxes can be integrated to any model isn't it ?), you can have your visual page builder to create the content of your model, what is the problem with that solution ?
